### DISCUSSION TITLE
Cloud commands?

### DISCUSSION THREAD

**Discussion by ghost** [mrbungle64/ecovacs-deebot.js#331] [Q&A]
Is there a way to control the deebot through Ecovacs cloud? As I ecovas-deebot.js is meant to control a deebot that is on the local LAN (AFAIK) but I need to be able to control on another location.

Is that (maybe with another package) at all possible?

**Comment by mrbungle64** [comment]
@cactusss 

This library (ecovas-deebot.js) is using the Ecovacs cloud 😉

**Reply by ghost** [reply]
Oh, thats great news (to me) didn't realise that. I always thougth my app was directly communication with my deebot. Thanks, that gives me new possibilities.

**Comment by mrbungle64** [comment]
You have to use something like [bumper](https://github.com/bmartin5692/bumper) (as a self-hosted central server) or [ecovacs-stack](https://github.com/m8schmit/ecovacs-stack) (frontend+backend) to control the Deebots directly without using the Ecovacs cloud.
